app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/components/MainScaffold.kt

```diff
@@ -57,13 +57,16 @@ private fun MainScaffold(
     navigateToSelectedItem: (MainNavigation) -> Unit,
     content: @Composable () -> Unit,
 ) {
-    AppNavigationSuiteScaffold(navType, NavigationItem.entries, currentDestination, navigateToSelectedItem) {
-        if (navType == NavigationSuiteType.None) {
-            content()
-        } else {
-            Column {
-                Box(modifier = Modifier.weight(1.0f)) { content() }
-                HorizontalDivider()
+    Column {
+        if (navType == NavigationSuiteType.NavigationRail) BrandTobAppBar()
```

Comment:
If there no way to have BrandTobAppBar() defined in the same place instead of on two separate and distant places depending on the NavigationSuiteType?

Reply:
We couldn't find one.

Reply:
To avoid it being too hidden, it can be exposed.
All we'd have to do is define a callback in our `MainScaffold` and call it `tabletTopBar`, for example, so that it's visible in our `MainScreen`.